src/main/java/com/openshift/jenkins/plugins/pipeline/Auth.java

```diff
@@ -49,7 +49,16 @@ private Auth(Collection<X509Certificate> certs, TaskListener listener,
     public static Auth createInstance(TaskListener listener, String apiURL,
             Map<String, String> env) throws RuntimeException {
         Auth auth = null;
-        File f = new File(CERT_FILE);
+        File f = null;
+        // first see if customer override cert file location
+        String certFile = env.get("CA_CERT_FILE");
+        if (certFile != null && certFile.trim().length() > 0) {
+            f = new File(certFile.trim());
+        }
+        // if did not override, or if provided bad value, use default
```

Comment:
i'm not sold on falling back to the default if the file doesn't exist.  at a minimum something needs to be logged if we do that.